### PR TITLE
Update run's changelog when a status is changed

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-1187

### CODE REVIEW

File: sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql [hunk 1]
```diff
@@ -312,23 +312,22 @@ CREATE TRIGGER StatusChange BEFORE UPDATE ON Status
...
+      INSERT INTO RunChangeLog(runId, columnsChanged, userId, message) SELECT
         runId,
         COALESCE(CONCAT_WS(',',
           CASE WHEN NEW.health <> OLD.health THEN 'health' END,
           CASE WHEN (NEW.completionDate IS NULL) <> (OLD.completionDate IS NULL) OR NEW.completionDate <> OLD.completionDate THEN 'completionDate' END,
           CASE WHEN (NEW.startDate IS NULL) <> (OLD.startDate IS NULL) OR NEW.startDate <> OLD.startDate THEN 'startDate' END,
           CASE WHEN NEW.runName <> OLD.runName THEN 'runName' END), ''),
-        (SELECT lastModifier FROM Run WHERE alias = NEW.runName),
-        log_message);
+        lastModifier,
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql]: Where does `lastModifier` come from?

Comment (apmasell) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql]: The run itself.

Comment (callunity) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql]: Ah, I missed that it had changed from `INSERT VALUES` to `INSERT SELECT`.

**Comment by callunity** [comment]
statusupdate

The `SELECT [...] lastModifier` doesn't seem to be behaving as expected.

**Comment by apmasell** [comment]
I'm not sure there's a sensible way to solve this without adding a last modifier field to the status or rolling status into run. Hibernate is updating the status before the run and I can't change that.

**Comment by callunity** [comment]
Are you sure that's what it is? I have updated the description and saved, and then updated the description and saved again, and it's still showing both changes as coming from the notification server. This indicates to me that the `run.setLastModifier` call in the `EditRunController` is not working properly.

**Comment by apmasell** [comment]
Oh, well, that's even better.

**Comment by apmasell** [comment]
Fixed.

**Pull request merged**